[2024-01-26T12:56:07.006+0000] {processor.py:157} INFO - Started process (PID=37) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:56:07.013+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:56:07.019+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:56:07.018+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:56:07.205+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:56:07.185+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:56:07.211+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:56:07.309+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.317 seconds
[2024-01-26T12:56:38.077+0000] {processor.py:157} INFO - Started process (PID=53) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:56:38.222+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:56:38.227+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:56:38.227+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:56:38.270+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:56:38.260+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:56:38.276+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:56:38.336+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.269 seconds
[2024-01-26T12:57:15.837+0000] {processor.py:157} INFO - Started process (PID=70) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:57:15.863+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:57:15.880+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:57:15.878+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:57:16.005+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:57:15.980+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:57:16.039+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:57:16.257+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.471 seconds
[2024-01-26T12:57:51.590+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:57:51.612+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:57:51.630+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:57:51.629+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:57:51.718+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:57:51.701+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:57:51.725+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:57:51.941+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.396 seconds
[2024-01-26T12:58:22.187+0000] {processor.py:157} INFO - Started process (PID=102) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:58:22.193+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:58:22.200+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:58:22.199+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:58:22.228+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:58:22.221+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:58:22.232+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:58:22.295+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.122 seconds
[2024-01-26T12:58:53.488+0000] {processor.py:157} INFO - Started process (PID=118) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:58:53.492+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:58:53.495+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:58:53.495+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:58:53.513+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:58:53.509+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:58:53.517+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:58:53.556+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.075 seconds
[2024-01-26T12:59:25.061+0000] {processor.py:157} INFO - Started process (PID=134) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:59:25.065+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:59:25.069+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:59:25.069+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:59:25.103+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:59:25.091+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:59:25.109+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:59:25.377+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.327 seconds
[2024-01-26T12:59:55.683+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:59:55.689+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T12:59:55.694+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:59:55.693+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:59:55.725+0000] {logging_mixin.py:149} INFO - [2024-01-26T12:59:55.718+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T12:59:55.729+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T12:59:55.790+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.116 seconds
[2024-01-26T13:00:26.468+0000] {processor.py:157} INFO - Started process (PID=166) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:00:26.475+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:00:26.481+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:00:26.481+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:00:26.531+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:00:26.521+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:00:26.543+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:00:26.643+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.191 seconds
[2024-01-26T13:00:56.938+0000] {processor.py:157} INFO - Started process (PID=182) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:00:56.942+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:00:56.945+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:00:56.944+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:00:56.962+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:00:56.958+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:00:56.964+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:00:57.004+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.074 seconds
[2024-01-26T13:01:27.397+0000] {processor.py:157} INFO - Started process (PID=198) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:01:27.400+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:01:27.403+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:01:27.402+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:01:27.426+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:01:27.413+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:01:27.428+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:01:27.466+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.076 seconds
[2024-01-26T13:01:58.709+0000] {processor.py:157} INFO - Started process (PID=214) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:01:58.712+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:01:58.715+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:01:58.714+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:01:58.733+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:01:58.729+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:01:58.738+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:01:58.773+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.071 seconds
[2024-01-26T13:02:29.312+0000] {processor.py:157} INFO - Started process (PID=230) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:02:29.315+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:02:29.318+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:02:29.318+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:02:29.335+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:02:29.330+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:02:29.337+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:02:29.370+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.065 seconds
[2024-01-26T13:02:59.568+0000] {processor.py:157} INFO - Started process (PID=246) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:02:59.572+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:02:59.576+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:02:59.575+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:02:59.594+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:02:59.590+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:02:59.596+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:02:59.629+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.068 seconds
[2024-01-26T13:03:30.210+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:03:30.214+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/dag_with_minio_s3.py for tasks to queue
[2024-01-26T13:03:30.219+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:03:30.218+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:03:30.240+0000] {logging_mixin.py:149} INFO - [2024-01-26T13:03:30.235+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dag_with_minio_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_minio_s3.py", line 4, in <module>
    from airflow.providers.amazon.aws.sensors.s3_key import S3KeySensor
ModuleNotFoundError: No module named 'airflow.providers.amazon.aws.sensors.s3_key'
[2024-01-26T13:03:30.246+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_minio_s3.py
[2024-01-26T13:03:30.293+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_with_minio_s3.py took 0.091 seconds
